pkg/util/event_emmiter_test.go

```diff
@@ -140,6 +147,29 @@ var _ = Describe("", func() {
 			})
 
 		})
+
+		It("should not update resource if it's nil", func() {
+			ee.Init(nil, nil, recorder)
+
+			By("should not emmit event for all three resources", func() {
```

Comment:
typo: emmit --> emit
same as in line 167

Reply:
Fixed.